hashicorp/consul
# Cc 7147 link to hcp modal

### Description
https://hashicorp.atlassian.net/browse/CC-7147
Add Link to HCP modal and integrate to link to hcp banner and side-nav item



https://github.com/hashicorp/consul/assets/10027860/1ce346bb-26df-4ce1-b639-0fb1fef70047

### Testing & Reproduction steps
1. go to Consul 
2. visit Services page
3. click purple banner 'Link the cluster' action
4. enjoy the modal, check how token is generated and try to link to hcp
5. open modal by clicking 'Link to HCP Consul Central' from side nav
6. repeat steps  



### Links



### PR Checklist
